backend/src/test/java/gov/cdc/usds/simplereport/db/repository/PersonRepositoryTest.java

```diff
@@ -12,12 +13,18 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.domain.Specification;
 
 class PersonRepositoryTest extends BaseRepositoryTest {
 
   @Autowired private PersonRepository _repo;
   @Autowired private OrganizationRepository _orgRepo;
 
+  private Specification<Person> inWholeOrganizationFilter(Organization org) {
```

Comment:
the fact that you need to recreate this here makes me feel like this `Specification` should be something you can retrieve from the `PersonRepository` or `Person` class itself. maybe it can take an organization as an arg, and the other copy of this in `PersonService` can pass in the current org as the arg. just spitballing, but you're effectively recreating the logic of something you're trying to test, which suggests that the design of the `PersonRepository` class needs a tweak.

Reply:
@benwarfield-usds Opinion?
I don't disagree, but I'm not sure where specification building should live. It seemed like the service layer was kind of right after I tried it down in the `PersonRepository`. 

It just seemed to complicate the model... But I agree that it does have a DB feel to to the `.and()` and `.or()`

Either way, moving it in a refactor will be pretty easy if we change our minds.

Reply:
I've only made heavy use of Specification-building once, and in that instance I built an entire service just to build Specifications. It was a very search-heavy tool. 

I don't think that this should be moved—the point here is that we're smoke-testing the specification-executing feature of the repository, which is mostly outside of our scope to test, so we just need to produce _some_ valid query to throw at it. It doesn't even have to be the same as one we use in PersonService! Though obviously we use those for a reason, so...